README.md

```diff
@@ -23,8 +23,8 @@ automatically updated when the git repository is committed to.*
 ## Acknowledgements
 
 The authors gratefully acknowledge Jessica Clarke (University of Cambridge),
-George Neville-Neil (MSB Associates), and Konrad Witaszczy (University of
-Cambridge)
+George Neville-Neil (MSB Associates), and Konrad Witaszczyk (University of
+Cambridge).
```

Comment:
Can I suggest merging this and a few other isolated, very minor changes (e.g., “ABI” after “hybrid” in one place) as a separate immediate commit, so that this pull request is just about the new package content?

Reply:
Sure, I added a separate PR for other changes: https://github.com/CTSRD-CHERI/cheribsd-getting-started/pull/5 .